singer-io/tap-stripe
# TDL 15120 add event type field in schemas

# Description of change
- Added **_updated_by_event_type_** field in schemas of following streams.
   - `charges`, `customers`, `disputes`, `invoice_items`, `invoices`, `payouts`, `products`, `plan`, `subscriptions`, `transfer`
- Updated `test_event_updates` and `test_all_fields` test cases.

# Manual QA steps
 - Run discover mode and verify that **_updated_by_event_type_** field is available in schemas and metadata of respective stream in the catalog.
 - Run sync mode and verify that **_updated_by_event_type_**  field is available in all records emitted by event_updates.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
